File: reference/CodeExamples/javaAppExamples/Android/src/com/db4odoc/android/Pilot.java
/* Copyright (C) 2004 - 2007 db4objects Inc. http://www.db4o.com */
package com.db4odoc.android;

public class Pilot {    
    private String name;
    private int points;  
    
    public Pilot(){
    	
    }
    
    public Pilot(String name,int points) {
        this.name=name;
        this.points=points;
    }
        
    public int getPoints() {
        return points;
    }
    
    public void addPoints(int points) {
        this.points+=points;
    }
    
    public String getName() {
        return name;
    }
    
    public String toString() {
        return name+"/"+points;
    }

	public void setName(String name) {
		this.name = name;
	}

	public void setPoints(int points) {
		this.points = points;
	}
}

File: db4o.net/Db4oTutorial/Db4odoc.Tutorial.Chapters/F1/Chapter6/Pilot.cs
namespace Db4odoc.Tutorial.F1.Chapter6
{
    public class Pilot
    {
        string _name;
        int _points;
        
        public Pilot(string name, int points)
        {
            _name = name;
            _points = points;
        }
        
        public int Points
        {
            get
            {
                return _points;
            }
        }
        
        public void AddPoints(int points)
        {
            _points += points;
        }
        
        public string Name
        {
            get
            {
                return _name;
            }
        }
        
        override public string ToString()
        {
            return string.Format("{0}/{1}", _name, _points);
        }
    }
}